README.md

```diff
@@ -17,3 +20,29 @@ graph TD;
     C-->D;
 ```
 ~~~
+
+## Mermaid
+
+Currently supporting Mermaid version 8.4.0.
+
+## Add custom CSS support
+
+You can use the built-in functionality to add custom CSS.
+
+For example: **Font Awesome**  
```

Comment:
Just link to existing [`markdown.styles` documentation](https://code.visualstudio.com/Docs/languages/markdown#_using-your-own-css) and show the suggested setting value itself:

```
"markdown.styles": [
    "https://use.fontawesome.com/releases/v5.7.1/css/all.css"
]
```

Reply:
resolved